packages/connectivity/connectivity/example/test_driver/test/integration_test.dart

```diff
@@ -2,10 +2,6 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-// TODO(amirh): Remove this once flutter_driver supports null safety.
-// https://github.com/flutter/flutter/issues/71379
-// @dart = 2.9
```

Comment:
We still need this for stable. We haven't decided how to do integration testing yet.

Reply:
Ok moving back, did we not say we are disabling stable integration test? Maybe I heard it wrong.

Reply:
The web plugins are doing this because apparently they already aren't running in CI, so there's essentially no downside.

For everything else, we need to figure out how we want to handle this in the period between now and the next stable release after 1.26.